File: Dataset/Leetcode/test/1/177.java
 class Solution {
    public int[] XXX(int[] nums, int target) {
        int[] result=new int[2];
        boolean flag=true;
        for (int i = 0; i < nums.length && flag; i++) {
            for (int j = 0; j < nums.length; j++) {
                if(i==j){
                    continue;
                }
                if(nums[i]+nums[j]==target){
                    result[0]=i;
                    result[1]=j;
                    flag=false;
                    break;
                }
            }
        }
        return result;
    }
}

File: Dataset/Leetcode/train/55/177.cpp
class Solution {
public:
    bool XXX(vector<int>& nums) {

        int n = nums.size();
        vector<bool> arrive(n, 0);

        if( n <= 1 )
            return true;
        arrive[0] = true;

        for( int i = 1 ; i < n ; i ++ ){
            for( int j = i - 1; j >= 0; j -- ){

                if( arrive[j] && nums[j] >= (i-j) ){
                    arrive[i] = true;
                    break;
                }

            }

        }

        return arrive[n-1];



    }
};


